posix/subsystem/src/epoll.cpp

```diff
@@ -273,8 +278,25 @@ struct OpenFile : File {
 							<< " is active" << std::endl;
 
 				// Abort early (i.e before requeuing) if the item is not pending.
```

Comment:
This comment is outdated.
```suggestion
				// Return pending items to the caller.
```